app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -137,7 +195,9 @@ class StateFragmentTest {
   fun testStateFragment_loadExp_explorationLoads_changeConfiguration_buttonIsNotVisible() {
     launchForExploration(TEST_EXPLORATION_ID_2).use {
       startPlayingExploration()
-      onView(isRoot()).perform(orientationLandscape())
+
+      rotateToLandscape()
```

Comment:
Not related to this PR but as this comes up here, I was thinking if we can have some generalised way of using things like this, as rotation is something which is widely used in testing.

Reply:
Agreed. It might actually be nice to extract more of the operations being done in this test to a general suite to help drive flows throughout the app. I actually like the emerged patterns here for robust test functions (e.g. scroll to the thing you want to interact with, interact with it, then wait for it to finish all-in-one).